charts/coralogix-operator/templates/secret.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if and (.Values.secret.create) (not .Values.secret.secretKeyReference) -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "coralogixOperator.fullname" . }}
+  {{- with .Values.secret.labels }}
+  labels:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with .Values.secret.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+type: Opaque
+stringData:
+  CORALOGIX_API_KEY: {{ .Values.secret.data.apiKey }}
```

Comment:
CORALOGIX_REGION is also required no?

Reply:
since it's not a sensible value I didn't require coming from the secret.
But if you think it's better have everything on the secret I can change to.

Reply:
Whatever you think makes more sense